protocols/imap/src/main/java/org/apache/james/imap/ImapSuite.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.james.imap;
```

Comment:
License missing

Reply:
Thanks for forcing push without looking against the fixup I pushed yesterday, now this issue is back. Please fix it then